atom/browser/api/atom_api_desktop_capturer.h

```diff
@@ -17,6 +19,12 @@ namespace api {
 class DesktopCapturer: public mate::EventEmitter<DesktopCapturer>,
                        public DesktopMediaListObserver {
  public:
+  struct Source {
+    DesktopMediaList::Source media_list_source;
+    // Will be an empty string if not available.
+    std::string screen_api_id;
```

Comment:
Perhaps just `screen_id`? Or `display_id`, to match the name of the `Display` data type in the screen API?

Reply:
I debated this myself. Changed to `display_id`.